content/ecosystem/_index.html

```diff
@@ -0,0 +1,444 @@
+---
+title: RAPIDS.ai | Ecosystem
+description: RAPIDS libraries, frameworks, extensions and more
+---
+
+  <!-- Hero Shortcode -->
+  {{< hero hero-text="Ecosystem" button-link="#software" button-text="Go to Software" >}}
+
+  <!-- Intro -->
+  <div id="hardware"></div>
+  <section class="section mb-6">
+    
+    <div class="container">
+      <h1 class="has-text-centered mb-5"> An ecosystem of hardware, <br> software, and developers</h1>
+      <div class="columns">
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-microchip"></i> Hardware </h2>
+          <p> RAPIDS runs on hardware powered by CUDA, the well-developed set of tools for GPU computing built by NVIDIA. RAPIDS works on PCs, workstations, data centers and the cloud. Learn more about the different types of hardware and find the best fit for your use case on NVIDA's product page for <a href="https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/"> Data Science Hardware <i class="fa-solid fa-arrow-up-right"></i></a> </p>
```

Comment:
Wording suggestion, typo fix, Oxford comma.
```suggestion
          <p> RAPIDS runs on hardware powered by CUDA, the fundamental set of tools for GPU computing built by NVIDIA. RAPIDS works on PCs, workstations, data centers, and the cloud. Learn more about the different types of hardware and find the best fit for your use case on NVIDIA's product page for <a href="https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/"> Data Science Hardware <i class="fa-solid fa-arrow-up-right"></i></a> </p>
```